Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.8
+
+WORKDIR /opt/app
+
+COPY . /opt/app
+
+RUN pip3 install -r requirements.txt
+
+ENTRYPOINT ["python", "./main.py"]
```

Comment:
Is there a need for having a `./main.py` or can it be just `main.py` as it is already present in the current working directory? Also, please change `python` to `python3` as the code is not supposed to be backwards-compatible to Python 2.

Reply:
Yes, It can be just main.py and python3 commands. I have updated it, please check.